Documentation/benchmarks/etcd-2-2-0-benchmarks.md

```diff
@@ -0,0 +1,67 @@
+# Physical Machines
+
+GCE n1-highcpu-2 machine type
+
+- 1x dedicated local SSD mounted as etcd data directory
+- 1x dedicated slow disk for the OS
+- 1.8 GB memory
+- 2x CPUs
+
+# etcd Cluster
+
+3 etcd 2.2.0 members, each runs on a single machine.
+
+Detailed versions:
+
+```
+etcd Version: 2.2.0
+Git SHA: e4561dd
+Go Version: go1.5
+Go OS/Arch: linux/amd64
+```
+
+# Testing
+
+Bootstrap another machine and use benchmark tool [boom](https://github.com/rakyll/boom) with customized patch to send requests to each etcd member. Check [here](../../hack/benchmark/) for instructions.
```

Comment:
Bootstrap another machine, outside of the etcd cluster, and run the [`boom` HTTP benchmark tool](https://github.com/rakyll/boom) with our connection reuse patch to send requests to each etcd cluster member. See the [benchmark instructions](../../hack/benchmark/) for the patch and the steps to reproduce our procedures.